### PR TITLE
nl-NL (Dutch) update

### PR DESCRIPTION
**Pull request by JappaWakka**
Tweaked the Dutch localization a bit

### CODE REVIEW

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
Fix line breaks in all files.

The commit does not have to update absolutely all rows.

**Comment by JappaWakka** [comment]
> Fix line breaks in all files.
> 
> The commit does not have to update absolutely all rows.

what needs to be fixed and how do I change the second part

**Comment by andrey-helldar** [comment]
@JappaWakka, see a screenshoots:

From me:

image

From you:

image

**Comment by andrey-helldar** [comment]
I replaced the commit

**Review by andrey-helldar — APPROVED** [review]

File: locales/nl/auth.php [hunk 1]
```diff
@@ -14,5 +14,5 @@
 return [
     'failed'   => 'Deze combinatie van e-mailadres en wachtwoord is niet geldig.',
     'password' => 'Het opgegeven wachtwoord is onjuist.',
-    'throttle' => 'Te veel mislukte loginpogingen. Probeer het over :seconds seconden nogmaals.',
-];
+    'throttle' => 'Te veel mislukte aanmeldpogingen. Probeer het over nog eens over :seconds seconden.',
+];
```
Comment (andrey-helldar) [inline locales/nl/auth.php:18]:
```suggestion
];

```

File: locales/nl/passwords.php [hunk 2]
```diff
@@ -17,4 +17,4 @@
     'throttled' => 'Gelieve even te wachten voor u het opnieuw probeert.',
     'token'     => 'Dit wachtwoordhersteltoken is niet geldig.',
     'user'      => 'Geen gebruiker bekend met het e-mailadres.',
-];
+];
```
Comment (andrey-helldar) [inline locales/nl/passwords.php:20]:
```suggestion
];

```

File: locales/nl/validation.php [hunk 3]
```diff
@@ -133,4 +133,4 @@
             'rule-name' => 'custom-message',
         ],
     ],
-];
+];
```
Comment (andrey-helldar) [inline locales/nl/validation.php:136]:
```suggestion
];

```

**Review by andrey-helldar — APPROVED** [review]

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
@WhereIsLucas, can you check this PR?

**Review by WhereIsLucas — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Dankjewel

**Pull request merged**